schedutils/chrt.c

```diff
@@ -184,29 +184,20 @@ static void show_sched_pid_info(struct chrt_ctl *ctl, pid_t pid)
 # endif
 	}
 
-	if (ctl->altered)
-		printf(_("pid %d's new scheduling policy: %s"), pid, get_policy_name(policy));
-	else
-		printf(_("pid %d's current scheduling policy: %s"), pid, get_policy_name(policy));
+	printf(_("pid %d's %s scheduling policy: %s"),
+		 pid, ctl->altered ? "new" : "current", get_policy_name(policy));
 
```

Comment:
The "new" and "current" should be translated (use _( ) macro). 

I doubt all the changes are translator-friendly because you hardcoded the word's position in the sentence.